### DISCUSSION TITLE
Resources not found

### DISCUSSION THREAD

**Discussion by hellohendri** [filamentphp/filament#1961] [Help] [answered]
Hello this is my first project using laravel and filament, it's going well on my local computer but the resources (app.css, app.js, livewire) is gone when I try to upload my project to the server. All of the links bellow returning 404 not found :

http://103.142.21.49/filament/assets/app.css?id=af3dd32f197fa0a6250d49f0a3c6e981
http://103.142.21.49/filament/assets/app.js?id=f7c71b9fce5b8f858591a0ef71c6e754
http://103.142.21.49/livewire/livewire.js?id=940557fc56b15ccb9a2d

Any help will be apreciated, thank you 🙏🏻

**Comment by ryangjchandler** [comment]
@hellohendri Is the APP_URL in your `.env` file the same as the actual URL you're accessing, i.e. `103.142.21.49`?

**Comment by danharrin** [comment, marked as answer]
This is probably a bad server (nginx or Apache config), make sure your server allows CSS / JS files to be returned from responses. It's a common issue in the Livewire community.

**Reply by hellohendri** [reply]
Thanks for the solution! I removed some script that maybe denied the  CSS / JS files from the nginx config and it worked! :)
